ui/src/translations/en.json

```diff
@@ -131,7 +131,10 @@
   "enter_label_name": "Enter label name",
   "enter_label_value": "Enter label value",
   "value": "Value",
-  "prometheus_not_available":"Prometheus is not available.",
-  "please_refer_to":"Please refer to",
-  "import_solution_hint":"You should import a solution to prepare your environment"
+  "prometheus_not_available": "Prometheus is not available.",
+  "please_refer_to": "Please refer to",
+  "import_solution_hint": "You should import a solution to prepare your environment",
+  "documentation": "Documentation",
+  "storage_class_creation_documentation": "storage class creation documentation",
+  "volume_creation_documentation": "volume creation documentation"
```

Comment:
Given that those icon are located next to the item they document, maybe `Access to the documentation`/`Accéder à la documentation` is enough.